opds.py

```diff
@@ -471,6 +471,7 @@ def groups(cls, _db, title, url, lane, annotator,
         """
         cached = None
         use_cache = cache_type != cls.NO_CACHE
+        facets = facets or lane.default_featured_facets(_db)
```

Comment:
Now that we're using `navigate` instead of the constructor we need to always have a faceting object to start with.